fec/draft-ietf-rtcweb-fec.xml

```diff
@@ -247,6 +249,12 @@
       e.g., by monitoring transmit packet loss data from RTCP Receiver Reports
       <xref target="RFC3550"/>), unless the application indicates it is willing
       to pay a quality penalty to proactively avoid losses.</t>
+      <t>Note that when probing bandwidth, i.e., speculatively sending
+      extra data to determine if additional link capacity exists, FEC
+      SHOULD be used in all cases. Given that extra data is going to be
+      sent regardless, it makes sense to have that data protect the
+      primary payload; in addition, FEC can be applied in a way that increases
+      bandwidth only modestly, which is necessary when probing.</t>
```

Comment:
@juberti citing "adaptive FEC" proposed as an informational reference might be relevant, the mechanism described in the draft can be applied by most congestion control algorithms and is shown to work. 

https://tools.ietf.org/html/draft-singh-rmcat-adaptive-fec-03